src/content/docs/core-concepts/volar-labs.mdx

```diff
@@ -1,18 +1,18 @@
 ---
-title: Volar Labs
+title: Volar.js Labs
```

Comment:
I changed this part because it was named Volar.js Labs on the VS Code Marketplace:

https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volarjs-labs

Please let me know if changing the name is a problem.

Reply:
I originally named it Volar.js Labs, but I agree to call it Volar Labs is better. :D

Reply:
Thanks for your comment! Then I will change it back to Volar Labs.